lib/cargoAudit.nix

```diff
@@ -1,22 +1,24 @@
 { cargoBuild
 , cargo-audit
+, vendorCargoDeps
 ,
-}: { cargoArtifacts
-   , cargoAuditExtraArgs ? ""
+}: { cargoAuditExtraArgs ? ""
    , cargoExtraArgs ? ""
    , advisory-db
    , ...
    } @ origArgs:
 let
   args = builtins.removeAttrs origArgs [ "cargoAuditExtraArgs" ];
 in
-cargoBuild (args
-  // {
-  inherit cargoArtifacts;
+cargoBuild (args // {
+  cargoVendorDir = args.cargoVendorDir or (vendorCargoDeps args);
+
```

Comment:
`cargoBuild` already does this for us automatically so we can omit this :)
```suggestion
```

Reply:
good point- it's gone :smile: